frontend/src/css/index.css

```diff
@@ -151,9 +151,18 @@ a, a:link, a:visited, a:hover, a:active {
 }
 
 .sliders-wrapper {
-  width: 300px;
-  height: 400px;
+  width: 70%;
+  height: 340px;
   display: flex;
   flex-direction: column;
   justify-content: center;
 }
+
+.feeling-wrapper {
+  width: 70%;
+  Height: 30%;
+  padding-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
Tiny change request here!... the explanation is long, but I swear the change is tiny 😅

We've been working towards dismantelling this file and using [CSS Modules](https://github.com/css-modules/css-modules). The reasoning behind this is that having a single file with plain classes makes it so that you can't use the same class name in a different place of the application (for example, imagine we now have to display something that would make total sense to call `.feeling-wrapper` in a completely different page to where you use this, and with very different rules).

Using CSS Modules allows us to, for example, use classes named `.wrapper` in multiple places with different styles, limiting a CSS file's scope to a single react component. This helps make the CSS more semantic, and it also means we don't need to remember every single class we ever wrote to avoid overlaps!

Just in case you're not sure how to use this based off what's already implemented, I'll leave a short guide now  but feel free to ask if you have any questions (both about how to do this and about why this is a cool choice!) 🙏 🙇

**When creating a new "view" component:**
1. Create a folder within the `routes` folder
2. Add an `index.js` and a `styles.module.css` (the `module.css` bit is the one that ensures our CSS Modules magic btw!)
3. Among the imports of the JS file, add:

```javascript
import styles from './styles.module.css';
```

4. Add any necessary CSS classes to the scoped, newly created CSS file
5. When you need to add the classes to the JSX code, instead of using plain strings (`className="wrapper"`) use the following: `className={styles.wrapper]}`

Happy coding!
![cute gif i've got](https://66.media.tumblr.com/366be7fd31e6f1e8c325c580d98ff99a/tumblr_n1guxnz7o41ri029no1_500.gif)